data/json/items/gun/762R.json

```diff
@@ -0,0 +1,112 @@
+[
+  {
+    "id": "mosin44",
+    "type": "GUN",
+    "name": "Mosin-Nagant M44",
+    "name_plural": "Mosin-Nagant M44",
+    "description": "The Model 44 is the carbine sized variant of the standard 1891/30.  It has a shorter barrel, therefore it is weaker than the 1891/30, but easier to carry.",
+    "weight": 4100,
+    "volume": 10,
+    "price": 23000,
+    "bashing": 12,
+    "to_hit": -1,
+    "material": [ "steel", "wood" ],
+    "symbol": "(",
+    "color": "brown",
+    "skill": "rifle",
+    "ammo": "762R",
+    "dispersion": 30,
+    "sight_dispersion": 60,
+    "aim_speed": 6,
+    "durability": 10,
+    "loudness": 150,
+    "clip_size": 5,
```

Comment:
Absolutely; a stripper clip's function is almost identical to a speedloader, though the particular geometry is different.

A stripper clip is basically a little rail that holds the bullets by the rims. When you want to load your gun all you have to do is slide it into the magazine port, and the bullets stay inside the magazine, while the stripper comes back out.

Granted, they don't work on every firearm with an internal magazine, the same as speedloaders only work on some revolvers (the sort where the cylinder breaks open or rolls out the side) but they can provide a massive boost in reloading speed when it counts.